srcpkgs/tiny/template

```diff
@@ -0,0 +1,18 @@
+# Template file for 'tiny'
+pkgname=tiny
+version=0.8.0
+revision=1
+build_wrksrc="tiny"
+build_style=cargo
+configure_args="--no-default-features --features=desktop-notifications --features=tls-native"
+hostmakedepends="pkg-config"
+makedepends="libressl-devel dbus-devel"
+short_desc="Tiny terminal IRC client"
+maintainer="eoli3n <jkirsz@gmail.com>"
+license="MIT"
+homepage="https://github.com/osa1/tiny"
+distfiles="https://github.com/osa1/tiny/archive/v${version}.tar.gz"
+checksum=acaf0b7c3515bdfd0c80b4a7274aec44d283025c6e40508e450525167f73e447
```

Comment:
Add a empty newline after checksum (just some informal convention to have a emtpy lines before a function)